pkg/virt-controller/services/template_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
```

Comment:
Duplicated library, `k8sv1` and `kubev1`

Reply:
Thank you @zhlhahaha. Will correct now